api/spec/controllers/spree/api/v1/checkouts_controller_spec.rb

```diff
@@ -260,6 +278,12 @@ module Spree
         expect_any_instance_of(PromotionHandler::Coupon).to receive(:apply).and_return({ coupon_applied?: true })
         api_put :update, :id => order.to_param, order_token: order.guest_token, order: { coupon_code: "foobar" }
       end
+
+      def send_request(params = {})
```

Comment:
Unused method argument - `params`. If it's necessary, use `_` or `_params` as an argument name to indicate that it won't be used. You can also write as `send_request(*)` if you want the method to accept any arguments but don't care about them.